### PR TITLE
fix cache bug

### PR DESCRIPTION
**Pull request by xithan**
includes delayed job cache fix

### CODE REVIEW

File: mod/calculated_metrics/set/type/metric/calculations.rb [hunk 1]
```diff
@@ -10,7 +10,9 @@
 end
 
 def calculation_in_progress!
-  Answer.where(id: all_dependent_answer_ids).update_all(calculating: true)
+  ids = all_dependent_answer_ids
+  Answer.where(id: ids).update_all(calculating: true)
+  Answer.where(id: ids).each(&:expire)
```
Comment (ethn) [inline mod/calculated_metrics/set/type/metric/calculations.rb]: Aren't these answer objects and not cards?

Comment (ethn) [reply mod/calculated_metrics/set/type/metric/calculations.rb]: Weird, Answer objects have an `expire` method, but I can't figure out where it's defined.

```
[5] pry(main)> Answer.first.method(:expire).source_location
=> nil
```

**Review by ethn — COMMENTED** [review]
Can you explain what `expire` is getting called?  It looks like both times it's called the expiration is on answer objects.

File: mod/calculated_metrics/set/type/metric/calculations.rb (hunk 1 above)
Comment (xithan) [reply mod/calculated_metrics/set/type/metric/calculations.rb]: Answer objects delegate missing method calls to the answer card object.

Comment (ethn) [reply mod/calculated_metrics/set/type/metric/calculations.rb]: Ahhhh.  thank you.

**Review by ethn — APPROVED** [review]

**Pull request merged**